digger/builds/__init__.py

```diff
@@ -22,8 +23,12 @@
     # then we can be sure that it is a Cordova project.
     # we also need to check if we want to build an IOS app or Android app for the Cordova.
     # for now, we don't do that and support Android only.
-    'is_valid': lambda path: os.path.exists('%s/config.xml' % path),
+    'is_valid': lambda path: os.path.exists('%s/www/config.xml' % path) or os.path.exists('%s/config.xml' % path),
     'builder': lambda path: CordovaAndroidBuild.from_path(path)
+  },
+  'cordovalight_android': {
+    'is_valid': lambda path: os.path.exists('%s/www/config.json' % path),
```

Comment:
Correct - but we will receive it as config.json for now (so better to leave it as it is), I think we can use the `.digger.yml` hidden file to specify the json plugin filename (default being digger.json). wdyt?